tests/local_mode/test_doc.py

```diff
@@ -108,3 +108,19 @@ def test_update_custom_attr_doc():
 
     # now check the updated attribute
     assert hasattr(doc._, "my_custom_tag") and doc._.my_custom_tag == "new_tag"
+
+
+def test_exclude_tokens_on_attr_values_doc():
+    """Test that the get_vector method ignores tokens based on the excluded_tokens dict"""
+    doc = nlp("Joey doesnt ever share food ")
+    doc_excluding_tokens = nlp("Joey doesnt share food")
+
+    # add custom_attr to the last token, the word ever
+    token = doc[2]
+    token.set_attribute(name="attribute1_name", value="value1")
+
+    # initialize excluded excluded_tokens dict
+    excluded_tokens = {"attribute1_name": ["value1", "value2"], "attribute2_name": ["v1", "v2"]}
```

Comment:
the word 'excluded' is repeated two time in your comment

Reply:
Np, I'll fix it in the next commit.